onnx_xla/onnx_xla_interface.cc

```diff
@@ -267,3 +277,79 @@ ONNXIFI_SYMBOL_NAME(onnxReleaseGraph)(onnxGraph graph) {
   }
   ONNXIFI_CATCH_EXCPETION();
 }
+
+//Initialize event by creating EventControl object on the heap
+ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI ONNXIFI_SYMBOL_NAME(
+    onnxInitEvent)(onnxBackend backend, onnxEvent* event) {
+  try {
+    if (!backend)  {
```

Comment:
There is no need to include these two checks into the try catch block

Reply:
@bddppq Is it not better to include for the specific return statuses - as described in onnxifi interface